app/common/model/src/main/java/io/syndesis/common/model/Schema.java

```diff
@@ -20,7 +20,7 @@
  */
 public final class Schema {
     // changing this will reset all the DB data.
-    public static final int VERSION = 40;
+    public static final int VERSION = 41;
```

Comment:
I don't think this needs changing, not without an `up-41.js` script, otherwise we might have an issue when someone adds `up-41.js` it will not be executed as the schema version is already at 41.

Reply:
But there is an up-41.js script.

Reply:
Missed that.